### ISSUE TITLE
[BUG] - (Enter your title here)

### ISSUE THREAD

**Issue by Mariozo** [tannerhelland/PhotoDemon#353]
### Version information
 - PhotoDemon version from Help > About, e.g. "PhotoDemon 8.4 latest stable release"
 - Windows Version (e.g. "Windows 10")

### Describe the bug
Probably can't be called a bug, PD link Help > About... > Participate in development... leads to 404 aka. get-involved :)

### How to reproduce the bug
The more detailed, the better!  If at all possible, a list works best, e.g.:
   1. Go to 'Help'
   2. Click on 'About...'

### Expected behavior
Perhaps the link should go to https://photodemon.org/participate/

### Thank you!

Please follow-up with this issue as I work on a fix.  As a "thank you", I'm happy to add your name to PhotoDemon's contributor list.  Just let me know what name and/or URL you'd like to use.

**Comment by tannerhelland** [comment]
Argh, this is embarrassing.  Thank you so much for catching this @Mariozo.  I made a typo in the redirect information from the old page to the new page at photodemon.org, from when I updated it earlier this year (to include a link to the new GitHub Discussions page for the project).  What a silly mistake 🤦‍♂️ 

Thank you again.  As noted in the bug report template, I'm happy to add your name to PhotoDemon's contributor list for the next release.  Just let me know what name and/or URL you'd like to use.
